store/tests/pmmr.rs

```diff
@@ -776,7 +776,7 @@ fn cleanup_rewind_files_test() {
 		&data_dir,
 		expected,
 		prefix_to_delete,
-		seconds_to_delete_after,
+		seconds_to_delete_after + 1,
```

Comment:
For those wondering why I had to do this:
This test create file such as they are removed if older than ```seconds_to_delete_after```. 
However this test is very fast so basically it does that.
1. Create file that are `seconds_to_delete_after` old.
2. Do some operation
3. Remove file that are older than `seconds_to_delete_after`.

However, on Windows CI (as you can see here https://dev.azure.com/qlesceller/qlesceller/_build/results?buildId=105&view=logs&jobId=75e8fe06-bfd6-5564-cb5c-dff4fd81606c&taskId=c33ab3a6-027f-5d3f-2c09-9b84aecf111c&lineStart=1608&lineEnd=1608&colStart=1&colEnd=9) this duration is in seconds thus the deletion never happens. 
```
let duration_since_accessed = now.duration_since(accessed)?;
  if duration_since_accessed <= cleanup_duration {
    return Ok(0); // these files are still too new
}
// Deletion happens here
```

Reply:
😑